settings/provisioner/openstack/openstack.spec

```diff
@@ -14,6 +14,18 @@ subparsers:
                   prefix:
                       type: Value
                       help: An prefix which would be contacted to each provisioned resource. An random prefix would be generated in case this value is not specified.
+            - title: key_file
+              options:
+                  key_file:
+                      type: Value
+                      help: The key file that would be uploaded to nova and injected into VMs upon creation.
+                      default: ~/.ssh/id_rsa
+            - title: key_name
+              options:
+                  key_name:
```

Comment:
If we are always deleting keys from openstack, it doesn't make sense for user to add his key here.
Let's remove `key_name` and always generate the keys from file.